test/opt/test_def_use.cpp

```diff
@@ -47,7 +52,9 @@ std::string DisassembleInst(ir::Instruction* inst) {
   std::string text;
   // We'll need to check the underlying id numbers.
   // So turn off friendly names for ids.
-  tools.Disassemble(binary, &text, SPV_BINARY_TO_TEXT_OPTION_NO_HEADER);
+  tools.Disassemble(binary, &text,
+                    SPV_BINARY_TO_TEXT_OPTION_NO_HEADER &
+                        ~SPV_BINARY_TO_TEXT_OPTION_FRIENDLY_NAMES);
```

Comment:
I'm surprised you have to explicitly exclude FRIENDLY_NAMES since it's not the default. ?